content/codesprint-2024/contributions.md

```diff
@@ -0,0 +1,163 @@
++++
+fragment = "content"
+weight = 40
+background = "light"
+
+#title = "GeoStyler Code Sprint 2023"
++++
+
+## Contributions
+
+During the Code Sprint, we achieved the following:
+
+<img src="/images/geostyler-codesprint-2024-stats.png" alt="GeoStyler Code Sprint Stats" />
+
+<br />
+
+### GeoStyler ArcGIS Parser
+
+One of the probably biggest and hopefully most impactful achievements is the
+introduction of the [geostyler-lyrx-parser](https://github.com/geostyler/geostyler-lyrx-parser) which allows
+for parsing of ArcGIS Pro styles (.lyrx format)! The parser is based on the [GeoCat/bridge-style](https://github.com/GeoCat/bridge-style)
+and was now ported from Python into TypeScript and integrated into the GeoStyler GitHub organisation, so that
+the parser integrates seamlessly with all the other existing parsers! That means that it is now possible to convert
+ArcGIS styles to QGIS, Mapbox, OpenLayers, and SLD! So far reading ArcGIS Pro styles with
+TODO list some capabilities here
+is supported.
+
+Kudos to 
+[@ger-benjamin](https://github.com/ger-benjamin),
+[@jwkaltz](https://github.com/jwkaltz),
+[@myriamfischer](https://github.com/myriamfischer),
+[@thomasmarti](https://github.com/thomasmarti),
+[@eschuh](https://github.com/eschuh),
+[@bsteinemann](https://github.com/bsteinemann).
+
+Besides that, an ArcGIS integration was created that allows for visually comparing ArcGIS styles with their SLD equivalents,
+making use of the [GeoStyler REST API](https://github.com/geostyler/geostyler-rest).
+
+Kudos to 
+[@myriamfischer](https://github.com/myriamfischer),
+[@thomasmarti](https://github.com/thomasmarti),
+[@eschuh](https://github.com/eschuh),
+[@bsteinemann](https://github.com/bsteinemann).
+
+### GeoStyler CLI
+
+Dependencies in the [geostyler-cli](https://github.com/geostyler/geostyler-cli) were upgraded to their latest versions. Furthermore,
+the executing commands for the different environments such as node, npx, binary executable, were harmonized. For all environments,
+the executing command will now be `geostyler-cli`!
+
+Kudos to
+[@geographika](https://github.com/geographika),
+[@jansule](https://github.com/jansule).
+
+### GeoStyler QGIS Plugin
```

Comment:
IMO, this could be the first topic of the blog. This feature, adding GeosStyler in a popular software, could offers a new visibility to the Geostyler project!